truffle-config.js

```diff
@@ -1,9 +1,6 @@
 module.exports = {
-//   compilers: {
-//     solc: '0.4.25'
-//   },
   networks: {
-    development: {
+    local: {
```

Comment:
really good to know about this, so Truffle will intuit that there's no ganache running and spin up a `ganache-cli` instance?

Reply:
Yes, this is a good way to let truffle provide an in memory ganache provider. It speeds things up. I modified the travis CI script to take advantage of that